packages/docsearch-react/src/__tests__/api.test.tsx

```diff
@@ -66,12 +68,13 @@ describe('api', () => {
           }}
         />
       );
+      // We can silence the potentially undefined here because we assert that they are not
```

Comment:
I don't think this comment is necessary, this is the TypeScript contract, we don't need to explain it in our codebase.